buildspec.yml

```diff
@@ -20,7 +20,7 @@ phases:
       - start_time=`date +%s`
       - |
         if has-matching-changes "tests/" "src/*.py" "setup.py" "setup.cfg" "buildspec.yml"; then
-          tox -e py36 -- tests/integ -m "not local_mode" -n 512 --reruns 3 --reruns-delay 5 --durations 50 --boto-config '{"region_name": "us-east-2"}'
+          tox -e py36 -- tests/integ -m "not local_mode" -n 512 --reruns 3 --reruns-delay 5 --durations 50 --boto-config '{"region_name": "us-east-1"}'
```

Comment:
FYI that we may need to revisit limits for the dev account for this change.

Reply:
Actually, almost certainly: [gw23] [ 52%] FAILED tests/integ/test_auto_ml.py::test_best_candidate

Reply:
i was hoping we had enough of everything in IAD, but seems like i was wrong. :(